src/controllers/bulk/bulkcontroller.cpp

```diff
@@ -130,6 +130,13 @@ bool BulkController::matchProductInfo(const ProductInfo& product) {
         return false;
     }
 
+#ifdef _WIN32
```

Comment:
```suggestion
#ifdef __WINDOWS__
```
At other places, we use our own definition __WINDOWS__ instead of _WIN32, which is correct of cause too.

Reply:
This got lost in the last in the last push. Better use indivitual commuts, to leep track.